.gitignore

```diff
@@ -2,6 +2,3 @@
 .vscode/
 .idea/
 target/
-examples/
-!examples/shopping_cart.rs
-!examples/taint/*.*
```

Comment:
Those exclusions actually don't work. From the doc: "It is not possible to re-include a file if a parent directory of that file is excluded." 

If we want to have a playground here it should be in a subdirectory or we . have a top-level 'playground'.

Reply:
Lines 6 and 7 can go away.  Line 5 can stay. You can include something into this directory by using `git add -f xxx`

Reply:
I did add everything manually with git add and then started to wonder why it was needed, and why the common -a workflow did not work. It took me a while to find the problem. I'm wondering why should the ./examples tree behave different than the others? Is this a well-known pattern I'm just not aware of?

Happy to change back, but don't understand the reasoning yet. As I said if you want to have a private playground in a repo tree, you could have a special subdir for that. I expect that examples grows over time, and will also serve as a set of integration tests.

Reply:
Well, examples IS the current private playground in the repo tree.

Reply:
Most projects on github and elsewhere use the 'examples' root directory of the repo for containing an authoritative set of usage examples of the sdk. Those are usually also part of the tests.

Because of this, it took me quite a while to consider looking in the .gitignore. I wasn't expecting a private playground in ./examples.

But never mind, we can address this later. Undone.